DESCRIPTION

```diff
@@ -10,12 +10,12 @@ Description: Provides easier interaction with
     format and manages throttling by 'Socrata'.
     Users can upload data to Socrata portals directly
     from R.
-Version: 1.7.1-24
-Date: 2016-11-04
+Version: 1.7.2-5
+Date: 2017-02-23
 Author: Hugh Devlin, Ph. D., Tom Schenk, Jr., and John Malc
 Maintainer: "Tom Schenk Jr." <developers@cityofchicago.org>
 Depends:
-    R (>= 3.0.0)
+    R (>= 3.1.0)
```

Comment:
@tomschenkjr I'm curious why this change incremented the dependency from R 3.0 to R 3.1